data/json/furniture.json

```diff
@@ -262,7 +262,7 @@
         "description": "Emergency water source, from the tank, and provider of relief.",
         "move_cost_mod": 2,
         "required_str": -1,
-        "flags": ["TRANSPARENT", "FLAMMABLE_HARD", "MOUNTABLE"],
+        "flags": ["TRANSPARENT", "FLAMMABLE_HARD", "MOUNTABLE", "LIQUIDCONT"],
```

Comment:
Why not `LIQUID_CONTAINER`?

Reply:
There was already a LIQUIDCONT flag, so no need to create another.